doc/content/xapi/guides/howtos/add-function.md

```diff
@@ -45,7 +45,7 @@ The parameters passed to call are all optional (except ~name and ~lifecycle).
 
 - The ~flags parameter is used to set conditions for the use of the message.
   For example, `Session is used to indicate that the call must be made in the
-  presence of an existing session.
+  presence of an existing session`.
```

Comment:
The backtick here is actually for the OCaml polymorphic variant

    `Session

so there shouldn't be one at the end.